### PR TITLE
Wrote documentation for 6 element endpoints

### PR DESCRIPTION
**Pull request by dpgraham**
Documented the following element endpoints

* /active
* /elements
* /element
* /value
* /submit
* /text

### CODE REVIEW

File: commands-yml/commands/element/find-element.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+name: Find Element
+short_description: Click on an element
+description:
+  >
+     Located element will be returned as a MobileElement JSON object.
+
+     The locator strategy returns the first element it finds. #TODO: Let's make a document with the locator strategies that this links to
```
Comment (jlipps) [inline commands-yml/commands/element/find-element.yml]: we talk about locator strategies in the new getting started doc, could link to that?

File: commands-yml/commands/element/find-element.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
+name: Find Element
+short_description: Click on an element
```
Comment (jlipps) [inline commands-yml/commands/element/find-element.yml]: this does not click on an element

File: commands-yml/commands/element/find-element.yml (hunk 1 above)
Comment (jlipps) [reply commands-yml/commands/element/find-element.yml]: we could also put the list of standard locator strategies here, and mention that drivers may support additional strategies?

Comment (jlipps) [reply commands-yml/commands/element/find-element.yml]: or if this is the one-stop shop for all drivers, we should list them in a table here I guess

File: commands-yml/commands/element/find-elements.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
+---
+name: Find Elements
+short_description: Search for multiple elements
+description:
+  >
+     The located elements will be returned as a list of MobileElement JSON objects.  #TODO: Let's make a document with the locator strategies that this links to
```
Comment (jlipps) [inline commands-yml/commands/element/find-elements.yml]: maybe we should reference the find element doc

File: commands-yml/commands/element/submit.yml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  java: "https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/WebElement.html#submit--"
+  python: "http://selenium-python.readthedocs.io/api.html#selenium.webdriver.remote.webelement.WebElement.submit"
+  javascript_wdio: "http://webdriver.io/api/action/submitForm.html"
+  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L1741"
+  ruby: "http://www.rubydoc.info/gems/selenium-webdriver/Selenium/WebDriver/" # TODO Ruby documentation link
+  php: "https://github.com/appium/php-client/" # TODO PHP documentation link
+  csharp: "https://github.com/appium/appium-dotnet-driver/" # TODO Dotnet documentation link
+
+# Driver support by platform
+driver_support:
```
Comment (jlipps) [inline commands-yml/commands/element/submit.yml]: i don't think this is supported unless you're in a webview context

File: commands-yml/commands/element/text.yml [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /wd/hub/session/:session_id/element/:element_id/text
+  method: 'GET'
+  url_parameters:
+    - name: session_id
+      description: ID of the session to route the command to
+    - name: element_id
+      description: ID of the element to route the command to
```
Comment (jlipps) [inline commands-yml/commands/element/text.yml]: i wouldn't say this command is being routed to an element. i'd just say it's the id of the element whose text we want.

File: docs/en/commands/element/find-element.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
+# Find Element
```
Comment (jlipps) [inline docs/en/commands/element/find-element.md]: why do we have both `element.md` and `find-element.md`? likewise with the multiple variant

File: docs/toc.js [hunk 7]
```diff
@@ -18,7 +18,13 @@ module.exports = {
     ["Commands", ["commands",
       ["Status", "status.md"],
       ["Element", ["element",
-        ["Click", "click.md"]
+        ["Click", "click.md"],
+        ["Active Element", "active.md"],
+        ["Find Element", "find-element.md"],
```
Comment (jlipps) [inline docs/toc.js]: I would put the finds at the very top

File: commands-yml/commands/element/active.yml [hunk 8]
```diff
@@ -0,0 +1,70 @@
+---
+name: Active Element
```
Comment (jlipps) [inline commands-yml/commands/element/active.yml]: "Active Element" doesn't sound like a command to me. maybe "Get Active Element"? or "Find Active Element"?

**Review by jlipps — COMMENTED** [review]
comments made!

File: commands-yml/commands/element/find-element.yml (hunk 1 above)
Comment (dpgraham) [reply commands-yml/commands/element/find-element.yml]: Yeah I think we should have a full table listing all of the strategies and a description for them and the support levels.

File: docs/en/commands/element/find-element.md (hunk 6 above)
Comment (dpgraham) [reply docs/en/commands/element/find-element.md]: That was a leftover artifact. The generation wasn't cleaning the folder. I'll add a `rimraf` to that.

File: docs/toc.js (hunk 7 above)
Comment (dpgraham) [reply docs/toc.js]: Yeah I think I'll add subcategories to elements as well. There's too many for one menu.

File: commands-yml/commands/element/actions/send-keys.yml [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+  javascript_wd:
+    |
+      let element = await driver.elementByAccessibilityId("SomeAccessibilityID");
+      await element.type("Hello world!")
+  javascript_wdio:
+    |
+      driver.setValue("~SomeAccessibilityId");
+  ruby:
+    |
+      @driver.find_element(:accessibility_id, "SomeAccessibilityID").send_keys("Hello World!"")
```
Comment (enriquesanchezb) [inline commands-yml/commands/element/actions/send-keys.yml]: I think there's an extra `"` in `"Hello World!""`

Comment (dpgraham) [reply commands-yml/commands/element/actions/send-keys.yml]: Good eye. Thanks!

**Pull request merged**